public/plugin.tsx

```diff
@@ -108,7 +116,8 @@ export class AssistantPlugin
           console.warn(`Action executor type ${actionType} is already registered.`);
         actionExecutors[actionType] = execute;
       },
-      assistantEnabled,
+      chatEnabled: () => this.config.chat.enabled,
+      userHasAccess: async () => await getAccount().then(checkAccess),
```

Comment:
The result of `userHasAcess` may not the same as the `HeaderChatButton` one. Shall we use the same getAccount result?

Reply:
they should be the same, account is cached